ctapipe/image/muon/tests/test_intensity_fit.py

```diff
@@ -20,52 +20,43 @@ def test_chord_length():
 
 
 def test_muon_efficiency_fit(prod5_lst, reference_location):
-    from ctapipe.coordinates import CameraFrame, TelescopeFrame
+    from ctapipe.coordinates import TelescopeFrame
     from ctapipe.image.muon.intensity_fitter import (
         MuonIntensityFitter,
         image_prediction,
     )
     from ctapipe.instrument import SubarrayDescription
 
+    radius = 1.1959596 * u.deg
```

Comment:
Overwritten before being used in l42 (with a slightly different value, 1.2 vs 1.1959596)